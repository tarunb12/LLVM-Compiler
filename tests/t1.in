int main() {
    // hello there
    printf("test")
}