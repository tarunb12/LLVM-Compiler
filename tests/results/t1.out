Syntax error: File "tests/t1.in", line 3: "@".